src/Angor/Client/Shared/NotificationComponent.razor

```diff
@@ -82,26 +97,32 @@
     {
         if (string.IsNullOrEmpty(message) && exception == null)
         {
-            return; // No error to handle
+            return;
         }
 
-        // Log the error with or without an exception
         if (exception != null)
         {
             Logger.LogError(exception, "An error occurred: {Message}", message ?? exception.Message);
+            exceptionDetails = $"Exception Type: {exception.GetType().Name}\n" +
```

Comment:
we are already logging errors to logs why do it again in the caller (like you did earlier)

Reply:
Yes, you're right, I removed it.